### DISCUSSION TITLE
Sign in both APK and bundle?

### DISCUSSION THREAD

**Discussion by gouri-panda** [Triple-T/gradle-play-publisher#1005] [Q&A] [answered]
Hi!, is there any way to sign in both APK and bundle? We need bundle for the play store and APK for with out play store.
We tried this approach
* Publish release bundle 
* upload to play store
* Publish release apk // We think it will be supposed to give us sign in apk format
Gives us this error
>A failure occurred while executing com.github.triplet.gradle.play.tasks.PublishApk$Processor
   > There was a failure while executing work items
      > A failure occurred while executing com.github.triplet.gradle.play.tasks.PublishApk$ApkUploader
         > 403 Forbidden
           {
             "code" : 403,
             "errors" : [ {
               "domain" : "global",
               "message" : "Cannot replace a bundle of version code 230405 with an APK.",
               "reason" : "forbidden"
             } ],
             "message" : "Cannot replace a bundle of version code 230405 with an APK.",
             "status" : "PERMISSION_DENIED"
           }

Is there any way to solve this problem?Thanks!

**Comment by SUPERCILEX** [comment, marked as answer]
Don't publish the APK. I believe `./gradlew assembleRelease` will build it into `build/outputs/apk/release`.

**Reply by gouri-panda** [reply]
Thanks! Do we need ssh_key or key_alias for this?

**Reply by SUPERCILEX** [reply]
Don't think so. If you've gotten signing to work with bundles, then it should already be working for APKs.
